v5/okapi/tutorials/walkthrough/autonomous-movement-async.rst

```diff
@@ -9,76 +9,63 @@ Oftentimes the fastest way to move in autonomous involves actuating multiple sub
+   const int DRIVE_LEFT_MOTOR_PORT = 1;
+   const int DRIVE_RIGHT_MOTOR_PORT = 2;
   
-   okapi::Motor driveMotor (DRIVE_MOTOR_PORT);
-   auto driveController = okapi::AsyncControllerFactory::posPID(driveMotor, drivekP, drivekI, drivekD);
+   auto driveController = okapi::ChassisControllerFactory::create(DRIVE_LEFT_MOTOR_PORT, DRIVE_RIGHT_MOTOR_PORT);
   
-And then we'll duplicate this to add a lift subsystem:
+And then we'll add a lift subsystem as an Async Controller as opposed to a Chassis Controller:
 
 .. highlight: cpp
 .. code-block:: cpp
    :linenos:
    :name: autonomous.cpp
 
-   const double drivekP = 1.0;
-   const double drivekI = 0.001;
-   const double drivekD = 0.1;
-   const int DRIVE_MOTOR_PORT = 1;
-  
+   const int DRIVE_LEFT_MOTOR_PORT = 1;
+   const int DRIVE_RIGHT_MOTOR_PORT = 2;
+   
    const double liftkP = 1.0;
    const double liftkI = 0.001;
    const double liftkD = 0.1;
    const int LIFT_MOTOR_PORT = 2;
   
-   okapi::Motor driveMotor (DRIVE_MOTOR_PORT);
-   auto driveController = okapi::AsyncControllerFactory::posPID(driveMotor, drivekP, drivekI, drivekD);
-  
-   okapi::Motor liftMotor (LIFT_MOTOR_PORT);
-   auto liftController = okapi::AsyncControllerFactory::posPID(liftMotor, liftkP, liftkI, liftkD);
+   auto driveController = okapi::ChassisControllerFactory::create(DRIVE_LEFT_MOTOR_PORT, DRIVE_RIGHT_MOTOR_PORT);
+
+   auto liftController = okapi::AsyncControllerFactory::posPID(LIFT_MOTOR_PORT, liftkP, liftkI, liftkD);
   
 Now that we have two subsystems to run, let's execute a few different movements. 
 
 If we want both systems to move, and the next movement in the autonomous routine only depends on the drive
 completing its movement (and it doesn't care about the lift's status), we'll run ``waitUntilSettled()``
 with just the drive controller.
 
-
 .. highlight: cpp
 .. code-block:: cpp
    :linenos:
    :name: autonomous.cpp
 
-   const double drivekP = 1.0;
-   const double drivekI = 0.001;
-   const double drivekD = 0.1;
-   const int DRIVE_MOTOR_PORT = 1;
-  
+   const int DRIVE_LEFT_MOTOR_PORT = 1;
+   const int DRIVE_RIGHT_MOTOR_PORT = 2;
+   
    const double liftkP = 1.0;
    const double liftkI = 0.001;
    const double liftkD = 0.1;
    const int LIFT_MOTOR_PORT = 2;
   
    void autonomous() {
-     okapi::Motor driveMotor (DRIVE_MOTOR_PORT);
-     auto driveController = okapi::AsyncControllerFactory::posPID(driveMotor, drivekP, drivekI, drivekD);
-  
-     okapi::Motor liftMotor (LIFT_MOTOR_PORT);
-     auto liftController = okapi::AsyncControllerFactory::posPID(liftMotor, liftkP, liftkI, liftkD);
+     auto driveController = okapi::ChassisControllerFactory::create(DRIVE_LEFT_MOTOR_PORT, DRIVE_RIGHT_MOTOR_PORT);
+
+     auto liftController = okapi::AsyncControllerFactory::posPID(LIFT_MOTOR_PORT, liftkP, liftkI, liftkD);
     
      // Begin movements
-     driveController.setTarget(1000); // Move 1000 ticks forward
+     driveController.moveDistanceAsync(1000); // Move 1000 ticks forward
```

Comment:
This is actually motor degrees, not ticks. The ChassisController's call `setEncoderUnits(AbstractMotor::encoderUnits::degrees)` in their ctors.